README.md

```diff
@@ -404,6 +445,9 @@ Registry has a `merge` function that enables you to expose multiple registries
 on the same endpoint. If the same metric name exists in both registries, an
 error will be thrown.
 
+Merging registries of different types is undefined. The user needs to make sure
+all used registries have the same type (Prometheus or OpenMetrics versions).
```

Comment:
again, can we throw a usweful error?

Reply:
https://github.com/siimon/prom-client/pull/482/files#diff-c76cf141522a047fee59ae738cbc8a0dc2c48ed15666d958d407d867c3a4b69fR237

Reply:
Docs should be updated to say it throws, no? Not undefined behavior